app/assets/javascripts/discourse/app/components/add-category-tag-classes.js

```diff
@@ -1,57 +1,51 @@
 import Component from "@ember/component";
-import { observes } from "discourse-common/utils/decorators";
 import { scheduleOnce } from "@ember/runloop";
 
 export default Component.extend({
-  _slug: null,
+  tagName: "",
+  currentClasses: null,
 
-  didInsertElement() {
-    this._super(...arguments);
-    this.refreshClass();
+  init() {
+    this.currentClasses = new Set();
+    this._super();
```

Comment:
You could convert this component to a native class to avoid having an init just for this one property. 🙂